### PR TITLE
feat: Added an object detection python sample app and new model

### PR DESCRIPTION
**Pull request by antoniomtz**
## PR Checklist

- [x ] Added label to the Pull Request for easier discoverability and search
- [ x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?
Added an object detection python sample app and new model

## Test Instructions if applicable

0. Build python apps: make build-python-apps
1. Download new video: cd benchmark-scripts && ./download_sample_videos.sh
2. Download new models: ./download_models/downloadOVMSModels.sh
3. Start camera simulator: ./camera-simulator/camera-simulator.sh
4. Run object detection pipeline: PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./docker-run.sh --workload opencv-ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
5. Observe result output files and video output.

### CODE REVIEW

**Review by vli11 — COMMENTED** [review]
running "object_detection" with good result, then I stopped it, and re-start pipeline run with "instance_segmentation", I've got errors.

File: configs/opencv-ovms/models/2022/config_template.json [hunk 1]
```diff
@@ -33,6 +33,17 @@
...
+            },
+            {"config": {
+                "name": "person_vehicle_bike_detection_2000",
+                "base_path": "/models/person_vehicle_bike_detection_2000/FP16-INT8",
+                "nireq": 1,
+                "batch_size":"1",
+		"shape": "auto",
+                "layout": "NHWC:NCHW",
+                "plugin_config": {"PERFORMANCE_HINT": "LATENCY"},
+                "target_device": "CPU"},
```
Comment (jim-wang-intel) [inline configs/opencv-ovms/models/2022/config_template.json:45]:
```suggestion
                "target_device": "{target_device}"},
```

File: download_models/downloadOVMSModels.sh [hunk 2]
```diff
@@ -65,10 +76,16 @@ getOVMSModelFiles() {
 
 if [ $segmentationModelDownloaded -eq 0 ]; then
     echo "download segmentation model..."
-    mkdir -p "$localSegmentationMdlDirName/FP16-INT8/1"
+    mkdir -p "$localPersonVehicleDetectionMdlDirName/FP16-INT8/1"
```
Comment (jim-wang-intel) [inline download_models/downloadOVMSModels.sh:79]: why change this? segmentation model should be the same right?

File: download_models/downloadOVMSModels.sh [hunk 3]
```diff
@@ -65,10 +76,16 @@ getOVMSModelFiles() {
...
     getOVMSModelFiles $segmentation $pipelineZooModel$segmentation $modelPrecisionFP16INT8 $localSegmentationMdlDirName
 fi
 
+if [ $personVehicleModelDownloaded -eq 0 ]; then
+    echo "download people vehicle model..."
+    mkdir -p "$localSegmentationMdlDirName/FP16-INT8/1"
```
Comment (jim-wang-intel) [inline download_models/downloadOVMSModels.sh:85]: should this be personVehicleModelDirName?  the variable naming is confusing...

File: download_models/downloadOVMSModels.sh (hunk 2 above)
Comment (antoniomtz) [reply download_models/downloadOVMSModels.sh:79]: good catch, I made a mistake here

File: download_models/downloadOVMSModels.sh (hunk 3 above)
Comment (antoniomtz) [reply download_models/downloadOVMSModels.sh:85]: yes

File: configs/opencv-ovms/models/2022/config_template.json (hunk 1 above)
Comment (antoniomtz) [reply configs/opencv-ovms/models/2022/config_template.json:45]: Should we use {target_device} or "MULTI:GPU,CPU" ? 
where does the value of target_device come from?

Comment (jim-wang-intel) [reply configs/opencv-ovms/models/2022/config_template.json:45]: it is the placeholder for the script to replace it upon user's input in command line

Comment (jim-wang-intel) [reply configs/opencv-ovms/models/2022/config_template.json:45]: you probably missed this PR from Brian: https://github.com/intel-retail/automated-self-checkout/pull/209  check it out

**Review by vli11 — APPROVED** [review]
the video and error log on segmentation was intermittently before your PR

**Review by jim-wang-intel — APPROVED** [review]

**Pull request merged**